project/core/templates/landing.html

```diff
@@ -1,172 +1,183 @@
+                        <button class="ripple">Login/SignUp</button>
+                    </a>
+                </nav>
             </div>
         </div>
-    </div>
-</header>
-<main>
-<div class="landing-content">
-    <div class="white" id="learnmore">
-        <div class="introduction full-height valign-wrapper">
-            <div class="container section center content-1 valign">
-                <div class="row">
-                    <div class="white cushion">
-                        <div class="container section">
-                            <h3 class="content-1-title bold">{% trans 'CiviWiki is a non-profit community working to develop a democratic engagement web system' %} .</h3>
-                            <div class="accent-line"></div>
-                            <p class="content-1-text section"> {% trans 'At CiviWiki we believe democracy works best when citizens are educated on issues and participate in the democratic process.' %}
-                            <p class="content-1-text section">We are a non-profit, non-partisan, community; working to build an open-source platform where citizens discuss the problems they face, discover solutions, and enact change.</p>
-                            </p>
-                        </div>
-                    </div>
-                </div>
+    </header>
+    <main>
+        <div class="landing">
+            <h2 class="landing-msg anim-typewriter">
+                <span>Discuss Problems.</span>
+                &nbsp;&nbsp;
+                <span>Discover Solutions.</span>
+                &nbsp;&nbsp;
+                <span>Enact Change.</span>
+            </h2>
+            <h2 class="landing-msg anim-typewriter">Building Democracy for a Better Age.</h2>
+            <div class="landing-msg btn-container">
+                <a href="/login/">
+                    <button class="ripple">JOIN CIVIWIKI</button>
+                </a>
+                <a href="/support_us/">
+                    <button class="ripple">LEARN MORE</button>
+                </a>
             </div>
         </div>
-        <div class="video full-height valign-wrapper">
-            <div class="container section valign">
-                <h4 class="section center white-text">{% trans 'Check out our short video describing CiviWiki' %} ...</h4>
-                <p class="center"></p>
-                <div class="videoWrapper">
-                    <iframe width="560" height="315" src="https://www.youtube.com/embed/Ll4VZv4a8bE" frameborder="0" allowfullscreen></iframe>
-                </div>
-                <div class="grey-text text-darken-3 center">{% trans 'Video made by' %}  <a class="grey-text text-darken-1" href="https://sheldonanim.com/" target="_blank">Jon Sheldon</a></div>
-            </div>
+        <div class="msg-1">
+            <h1 class="title">
+                CiviWiki is a non-profit community working to develop a democratic engagement web system.
```

Comment:
Add `translate` tag here
```suggestion
                {% translate "CiviWiki is a non-profit community working to develop a democratic engagement web system." %}
```